softgroup/services/authorization/authorization-api/src/main/java/com/softgroup/authorization/api/message/LoginDataReq.java

```diff
@@ -6,7 +6,7 @@
  * @author odin
  * @since 20.02.17.
  */
-public class LoginRequest implements RequestData{
+public class LoginDataReq implements RequestData{
```

Comment:
@VasjaVn LoginRequest is simple and clear. But if you like long names - than name it LoginRequestData. The same for other requests.